docs/source/get_involved/governance.rst

```diff
@@ -324,6 +324,12 @@ expected to resign voluntarily.
 Tenure also ends automatically when a CC member's tenure as core developer ends,
 e.g., through resignation or inactivity.
 
+CC members can be removed from the CC by an 80% majority of all core developers,
+i.e., non-participation abstention counts as against.
+This vote is anonymous and takes place on the private core developer channel.
+It requires a 5 working day (excl week-ends) discussion period, followed
```

Comment:
```suggestion
```

Reply:
I think this is important, people need to know this is coming. We have this for appointments, so we should also do this for revocations.